website/project/model.py

```diff
@@ -118,6 +120,31 @@ def requires_consent(self):
     def has_files(self):
         return self._config.get('hasFiles', False)
 
+    @property
+    def absolute_api_v2_url(self):
+        path = '/metaschemas/{}/'.format(self._id)
+        return api_v2_url(path)
+
+    # used by django and DRF
+    def get_absolute_url(self):
+        return self.absolute_api_v2_url
+
+    def validate_metadata(self, *args, **kwargs):
+        """
+        Validates registration_metadata field.
+        """
+        reviewer = kwargs.pop('reviewer', False)
```

Comment:
Better to explicitly put these in the method signature rather than pulling off of kwargs. This makes the code more readable and easier for static analysis (e.g. IDEs can understand the signature better).